src/Semantics/Attribute/ClientRequestAttributeProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the worldia/instrumentation-bundle package.
+ * (c) Worldia <developers@worldia.com>
+ */
+
+namespace Instrumentation\Semantics\Attribute;
+
+use OpenTelemetry\SemConv\TraceAttributes;
+
+class ClientRequestAttributeProvider implements ClientRequestAttributeProviderInterface
+{
+    /**
+     * @param array<string> $capturedHeaders
+     */
+    public function __construct(private array $capturedHeaders = [])
```

Comment:
Should we allow to capture the request body as well ?

Reply:
i dont think so... do you see a use case?